src/encoded/schemas/experiment_set.json

```diff
@@ -97,6 +97,13 @@
             "lookup": 20,
             "formInput": "textarea"
         },
+        "disease_name": {
+            "title": "Disease Name",
+            "description": "Ontology term for the disease name associated with the set.",
```

Comment:
I think the description is OK but maybe add 'disorder' as not everything is a 'disease' i.e. for the disease or disorder ...